src/sprite/mod.rs

```diff
@@ -200,6 +202,13 @@ impl Sprite {
         self.vertices = update_vertices(&self.texture, &self.props, &self.texture_transform);
     }
 
+    pub fn set_position(&mut self, pos: (f32, f32, f32)) {
```

Comment:
Seeing this I really am leaning even more toward ECS. Either way, not really sure about this with the position value being used over the actual Transform even though Transform is still being set. Need to really start cleaning all this up.